rasa_sdk/forms.py

```diff
@@ -269,7 +315,10 @@ def extract_requested_slot(
 
                 if mapping_type == "from_entity":
                     value = self.get_entity_value(
-                        requested_slot_mapping.get("entity"), tracker
+                        requested_slot_mapping.get("entity"),
+                        tracker,
+                        role=requested_slot_mapping.get("role"),
+                        group=requested_slot_mapping.get("group"),
```

Comment:
```suggestion
                        requested_slot_mapping.get("entity"),
                        tracker,
                        requested_slot_mapping.get("role"),
                        requested_slot_mapping.get("group"),
```
to be consistent with the other times you call this function